packages/babel-generator/src/generators/methods.ts

```diff
@@ -10,7 +10,6 @@ export function _params(
   this.token("(");
   this._parameters(node.params, node);
   this.token(")");
-  this._noLineTerminator = true;
```

Comment:
This is the real fix, can you confirm it? I'm not sure about the purpose here.
@nicolo-ribaudo

Reply:
In arrow functions, `)` cannot be followed by a newline.

Reply:
Probably it doesn't matter because inner comments in arrow functions would already be printer earlier.